build.gradle

```diff
@@ -46,4 +46,6 @@ jdeploy {
     description = "Android vector drawable to SVG"
     license = "Apache-2.0"
     repository = "https://github.com/neworld/vd2svg"
+    jar = file("$buildDir/libs/vd2svg-0.3.1.jar")
```

Comment:
Why do you need to specify jar? Does not `gradle-jdeploy-plugin` detect it automatically?

Reply:
Strange enough i got
```
'org.gradle.api.provider.Provider org.gradle.api.tasks.bundling.Jar.getArchiveFile()'
```

then tried:
```
jar = file("$buildDir/libs/vd2svg-$version.jar")
```
And got
```
* What went wrong:
A problem occurred evaluating root project 'vd2svg'.
> No such property: jar for class: java.lang.String
```

Reply:
Ok. Something had broken. Could you reset build configuration changes? I will check rest and make a release

Reply:
I agree that this is somehow a very unsatisfying solution

Reply:
Added the version into the jar line, let's see if this works

Reply:
okay will do